platform-cloud/docs/git/overview.md

```diff
@@ -175,7 +179,7 @@ To connect to a private BitBucket repository, see the [BitBucket documentation](
 
 3. Select _BitBucket_ as the **Provider**.
 
-4. Enter your **Username** and **Password**.
+4. Enter your **Username** (email) and **Token**.
```

Comment:
When using API tokens, the username is the account email. However, when using app passwords the username is the account nickname. Any suggestions on how to express it better, @llewellyn-sl?